src/constants/localStorage.ts

```diff
@@ -1,4 +1,3 @@
 export const localStorageKeys = {
-  notificationModalClosed: 'w3i:notification_modal_closed',
-  launchBannerClosed: 'w3i:launch_banner_closed'
```

Comment:
Hmm I guess we are abandoning this boolean in their storage forever!

Reply:
Yeap, no need to use again so removed the key